app/services/further_information_request_reminder.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+class FurtherInformationRequestReminder
+  include ServicePattern
+  include FurtherInformationRequestExpirable
+
+  def initialize(further_information_request:)
+    @further_information_request = further_information_request
+  end
+
+  def call
+    if send_reminder?
+      send_email
+      record_reminder
+      create_timeline_event
+    end
+  end
+
+  private
+
+  attr_reader :further_information_request
+
+  def send_reminder?
+    two_weeks? || one_week? || two_days?
+  end
+
+  def number_of_reminders_sent
+    further_information_request.reminder_emails.count
+  end
+
+  def two_weeks?
+    days_until_expiry <= 14 && number_of_reminders_sent.zero?
+  end
+
+  def one_week?
+    days_until_expiry <= 7 && number_of_reminders_sent == 1
+  end
+
+  def two_days?
+    days_until_expiry <= 2 && number_of_reminders_sent == 2
+  end
+
+  def send_email
+    TeacherMailer
+      .with(teacher:, further_information_request:)
+      .further_information_reminder(due_date:)
+      .deliver_later
+  end
+
+  def record_reminder
+    further_information_request.reminder_emails.create!
+  end
+
+  def create_timeline_event
+    TimelineEvent.create!(
+      application_form:,
+      creator_name: "Expirer",
+      mailer_action_name: "further_information_reminder",
+      event_type: "email_sent",
+    )
+  end
```

Comment:
We shouldn't need to do this as https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/main/app/lib/teacher_mailer_observer.rb will create this for us.